IM_calculation/source_site_dist/src_site_dist.py

```diff
@@ -218,6 +218,64 @@ def calc_rx_ry_GC2(
     return r_x, r_y
 
 
+def calc_rx_ry_GC2_multi_hypocentre(
+    srf_points: np.ndarray,
+    plane_infos: List[Dict],
+    locations: np.ndarray,
+    origin_offsets: np.ndarray = np.asarray([0]),
+):
```

Comment:
I think the existing function can be turned into a wrapper which calls this in the case of only one hypocentre. I don't think it makes sense to have two copies of almost the same function when it can be reduced to one copy.

Reply:
Is there a plan to turn one of these functions into a wrapper for the other (either way)? I don't think there's a need for both to have explicit implementations.